test/cql-pytest/test_cdc.py

```diff
@@ -20,8 +21,9 @@ def wait_for_first_cdc_generation(cql, timeout):
         assert time.time() < deadline, "Timed out waiting for the first CDC generation"
         time.sleep(1)
 
-# xfail_tablets due to https://github.com/scylladb/scylladb/issues/16317
-def test_cdc_log_entries_use_cdc_streams(scylla_only, cql, test_keyspace, xfail_tablets):
+# xfails with tablets, due to https://github.com/scylladb/scylladb/issues/16317
+@pytest.mark.parametrize("test_keyspace", [pytest.param("tablets", marks=[pytest.mark.xfail]), "vnodes"], indirect=True)
```

Comment:
Please use `pytest.mark.xfail(reason="issue #16317")` - the xfail reason is printed e.g., when running pytest -vvs.

Reply:
Fixed in v2.